iconfont/iconjar-map.js

```diff
@@ -0,0 +1,52 @@
+// load desired modules
+var fileSystem = require( 'fs' );
+var readLine = require( 'readline' );
+var path = require( 'path' );
+ 
+// find the filename we wish to use
+var name = null;
+var dirListing = fileSystem.readdirSync( __dirname );
+
+for( var i = 0; i < dirListing.length; i++ ) {		
```

Comment:
Since we'll be using this as a one-off for now, you may as well drop this search and bake in the name as a constant (since you're already doing the same with `codepoints`)